aroquev/.gitignore

```diff
@@ -1 +1 @@
-target/*
+*target/*
```

Comment:
You can put a .gitignore in a subdirectory and it will apply to that dir & its subdirectories only.